apps/studio/src/routes/editor/WebviewArea/GestureScreen.tsx

```diff
@@ -174,7 +174,7 @@ const GestureScreen = observer(({ webviewRef, setHovered }: GestureScreenProps)
             <div
                 className={cn(
                     'absolute inset-0 bg-transparent',
-                    editorEngine.mode === EditorMode.INTERACT ? 'hidden' : 'visible',
+                    editorEngine.mode === EditorMode.INTERACT ? 'visible' : 'visible',
```

Comment:
This is not what I meant at all. We don't want to keep it always visible because then interact mode does not work. It should be hidden during interact unless resizing is happening.

Reply:
Apologies for earlier. I misunderstood the requirements regarding the visibility during interact mode. I've now updated the feature so that the element is hidden during interact mode unless resizing is occurring, as you originally intended.

https://github.com/user-attachments/assets/9a90133f-880b-4756-93df-e3feb9e13851